test/micro/org/openjdk/bench/vm/compiler/overhead/SimpleRepeatCompilation.java

```diff
@@ -135,25 +135,25 @@ public int trivialMath(int a, int b) {
      = "-XX:CompileCommand=option,org/openjdk/bench/vm/compiler/overhead/SimpleRepeatCompilation.largeMethod,intx,RepeatCompilation,100";
 
     @Benchmark
-    @Fork(jvmArgsAppend={"-Xbatch",LARGE_METHOD})
+    @Fork(jvmArgs={"-Xbatch",LARGE_METHOD})
```

Comment:
```suggestion
    @Fork(jvmArgs={"-Xbatch", LARGE_METHOD})
```

Reply:
I don't think this PR is the place to address pre-existing and non-consequential style issues.